testkit/testkit.go

```diff
@@ -176,3 +177,14 @@ func newSession(t *testing.T, store kv.Storage) session.Session {
 	se.SetConnectionID(testKitIDGenerator.Inc())
 	return se
 }
+
+// MustGetErrCode executes a sql statement and assert it's error code.
+func (tk *TestKit) MustGetErrCode(sql string, errCode int) {
```

Comment:
This function doesn't used?

Reply:
used in `table/tables/partition_test.go` line 509